### PR TITLE
Extend length of status bars in status page

### LINKED ISSUE #4365: Could we extend the length of status bar?
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Could we extend the length of status bar? I think longer is more beautiful.

### ✔️ Solution

IMG_20240113_134519

### ❓ Alternatives

_No response_

### 📝 Additional Context

_No response_

### PR DESCRIPTION
**Pull request by amyyeung17**
Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

Fixes #4365. 
Adjusted column values in PublicGroupList to extend status bars in small screens. 

## Type of change

Please delete any options that are not relevant.
- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Screen Shot 2024-01-16 at 2 27 11 PM

## Comments
I wasn't sure if the request was asking to extend for all or just small screens. Based on the screenshot, I assumed the latter.

### CODE REVIEW

**Comment by mobeicanyue** [comment]
nice job sir, thank you.
I not familiar with vue, could you explain it for me?

 u change it from 
<div class="col-9 col-md-8 small-padding">
to
<div class="col-8 small-padding">

What is it function?

**Comment by mobeicanyue** [comment]
bro, I think this length will be better
```
<div class="col-6 col-md-4 small-padding">
```
```
<div :key="$root.userHeartbeatBar" class="col-6 col-md-8">
```
image

image

File: src/components/PublicGroupList.vue [hunk 1]
```diff
@@ -33,7 +33,7 @@
                         <template #item="monitor">
                             <div class="item">
                                 <div class="row">
-                                    <div class="col-9 col-md-8 small-padding">
+                                    <div class="col-8 small-padding">
```
Comment (mobeicanyue) [inline src/components/PublicGroupList.vue:36]: <div class="col-6 col-md-4 small-padding">

File: src/components/PublicGroupList.vue [hunk 2]
```diff
@@ -70,7 +70,7 @@
                                             </div>
                                         </div>
                                     </div>
-                                    <div :key="$root.userHeartbeatBar" class="col-3 col-md-4">
+                                    <div :key="$root.userHeartbeatBar" class="col-4">
```
Comment (mobeicanyue) [inline src/components/PublicGroupList.vue:73]: <div :key="$root.userHeartbeatBar" class="col-6 col-md-8">

**Review by mobeicanyue — COMMENTED** [review]
I think this style of bar will be better

image

image

**Review by CommanderStorm — APPROVED** [review]
Thanks for the style change! 🎉 



> [!NOTE]
> This PR is part of the [`v2.0` merge window](https://github.com/louislam/uptime-kuma/milestone/24) => see #4500 for the bugs that need to be addressed before we can ship this release ^^

<sub>All changes in this PR are small and uncontroversial ⇒ merging with junior maintainer approval</sub>

**Pull request merged**